.github/workflows/test_precommit.yml

```diff
@@ -171,24 +166,41 @@ jobs:
         set PATH=opencv\opencv\build\x64\vc16\bin;w_openvino_toolkit_windows_2023.0.0.10926.b4452d56304_x86_64\runtime\bin\intel64\Release;w_openvino_toolkit_windows_2023.0.0.10926.b4452d56304_x86_64\runtime\3rdparty\tbb\bin;%PATH%
         .\build\Release\test_sanity.exe -d data -p tests\cpp\precommit\public_scope.json && .\build\Release\test_model_config -d data
   serving_api:
-      strategy:
-        fail-fast: false
-        matrix:
-          python-version: [3.7, 3.8, 3.9, '3.10', '3.11']
-          os: [ubuntu-20.04, ubuntu-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04, macos-11, macos-12]
+        python-version: [3.7, 3.8, 3.9, '3.10', '3.11']
```

Comment:
too many versions of Python. The normal approach is to test only 3.7 and 3.11. what do you think?

Reply:
I'd say it should also include the default python version for a given OS in addition to lower and upper bounds.

Extending that list costs me only one entry in matrix, so I thought why not to extend it. If I notice that there are queues that delay overall build results, I will shrink the list.

Reply:
Ok, let it be. Now, it is not a bottleneck for CI.